examples/plot_simulate_gamma.py

```diff
@@ -33,30 +33,60 @@
 net = Network(params)
 dpls = simulate_dipole(net, n_trials=1)
 
+###############################################################################
+# The network requires some time to reach steady state. Hence, we omit the
+# first 50 ms in our time-frequency analysis.
+
+tstart = 50
+mask = dpls[0].times > tstart
+times = dpls[0].times[mask]
+data = dpls[0].data['agg'][mask]
```

Comment:
```suggestion
include_samples = dpls[trial_index].times > tstart
...
```

Reply:
Just to be explicit about what you're doing. Users may not feel comfortable with 'magic' constants like `0` before they get the hang of the objects.

Reply:
okay with adding trial_index. I actually prefer `mask` as opposed to include_samples though. It's easier to read the code because I know that it's a masked array as opposed to being indices or slices.